Minor additions to lay groundwork for features to be implemented

justin-obrien-smoothstack opened the pull request:
This PR should probably be merged before any of my others (except the DTO deannotation one, for which the order shouldn't matter).

justin-obrien-smoothstack committed:
Apply STS autoformatting to 3 service classes

AdminAuthorService, AdminBranchService, AdminPublisherService

justin-obrien-smoothstack committed:
Make skeleton of AdminAuthorController class

justin-obrien-smoothstack committed:
Change name of a method in AdminAuthorService

Change method name from updateAAuthor to updateAuthor

justin-obrien-smoothstack committed:
Add Jackson Dataformat XML dependency to pom.xml

justin-obrien-smoothstack committed:
Ignore & untrack application.properties

justin-obrien-smoothstack committed:
Merge remote-tracking branch 'origin/dev' into dev-jo

justin-obrien-smoothstack committed:
Untrack application.properties (again?)

justin-obrien-smoothstack committed:
Move & rename package for controller classes

justin-obrien-smoothstack committed:
Create skeleton of AdminPublisherController class

justin-obrien-smoothstack committed:
Rename AdminAuthorService object in AdminAuthorController class

justin-obrien-smoothstack committed:
Add text files for documenting manual tests

justin-obrien-smoothstack committed:
Add @RestController annotation to AdminPublisherController definition

justin-obrien-smoothstack committed:
Make skeleton of AdminBranchController class

sean-darsie reviewed (approved):
Make sure to add the resources/applications.properties to the .gitignore before the next pr.

justin-obrien-smoothstack committed:
Remove application.properties

Hopefully what I've done will exclude the file from future commits,
but allow me to keep the file around in my local working tree

justin-obrien-smoothstack commented:
It's there. I probably modified and committed it before I ignored it though. I've been having a bit of trouble trying to exclude it from commits and pushes while keeping it on my machine. Hopefully this last commit did the trick.
